requirements.txt

```diff
@@ -57,5 +57,8 @@ supervisor==3.3.1
 tzlocal==1.4
 uritemplate==0.6
 urllib3==1.22
-uwsgi==2.0.12
+uwsgi==2.0.17.1
 virtualenv==1.11.4
+moto==1.3.6
```

Comment:
We don't actually require moto anywhere, do we need that line?

Reply:
We will need it.  We're going to wrap the SQS tests and CloudTrail tests in moto mocks.